--- python-ldap-2.4.10/setup.cfg~	2012-06-07 21:41:34.000000000 +0300
+++ python-ldap-2.4.10/setup.cfg	2013-01-30 11:54:18.897363390 +0200
@@ -1,6 +1,5 @@
 # Define extra include and library dirs if needed
-library_dirs = /opt/openldap-RE24/lib /usr/lib
-include_dirs = /opt/openldap-RE24/include /usr/include/sasl /usr/include
+include_dirs = /usr/include/sasl
 
 # These defines needs OpenLDAP built with
 # ./configure --with-cyrus-sasl --with-tls
