diff -durN python-ldap-1.10alpha3.orig/Modules/LDAPObject.c python-ldap-1.10alpha3/Modules/LDAPObject.c
--- python-ldap-1.10alpha3.orig/Modules/LDAPObject.c	Tue Aug 15 00:37:37 2000
+++ python-ldap-1.10alpha3/Modules/LDAPObject.c	Sun Oct 21 17:16:59 2001
@@ -1421,89 +1421,6 @@
 
 /* ldap_search_s == ldap_search_st */
 
-/* ldap_ufn_search_c */
-
-/* ldap_ufn_search_ct */
-
-/* ldap_ufn_search_s */
-
-static PyObject*
-l_ldap_ufn_search_s( LDAPObject* self, PyObject* args )
-{
-    char *ufn;
-    PyObject *attrlist;
-    char **attrs;
-    int attrsonly = 0;
-    LDAPMessage *resmsg = NULL;
-    int result;
-
-    if (!PyArg_ParseTuple( args, "sO|i", 
-    	&ufn, &attrlist, &attrsonly)) return NULL;
-    if (not_valid(self)) return NULL;
-
-    if (!attrs_from_List( attrlist, &attrs )) 
-   	 return NULL;
-
-    LDAP_BEGIN_ALLOW_THREADS( self );
-    result = ldap_ufn_search_s( self->ldap, ufn,
-                             attrs, attrsonly, &resmsg );
-    LDAP_END_ALLOW_THREADS( self );
-
-    free_attrs( &attrs );
-
-    if (result != LDAP_SUCCESS)
-    	return LDAPerror( self->ldap, "ldap_ufn_search_s" );
-
-    if (resmsg == NULL) {
-    	Py_INCREF( Py_None );
-	return Py_None;
-    } else {
-    	return LDAPmessage_to_python( self->ldap, resmsg );
-    }
-}	
-
-
-/* ldap_ufn_setfilter */
-
-static PyObject*
-l_ldap_ufn_setfilter( LDAPObject* self, PyObject* args )
-{
-    char* filter;
-    LDAPFiltDesc* res;
-
-    if (!PyArg_ParseTuple( args, "s", &filter)) return NULL;
-    if (not_valid(self)) return NULL;
-    res = ldap_ufn_setfilter( self->ldap, filter );
-
-    if (res == NULL)
-	return LDAPerror(NULL, "ldap_ufn_setfilter");
-
-    Py_INCREF( Py_None );
-    return Py_None;
-}
-
-/* ldap_ufn_setprefix */
-
-static PyObject*
-l_ldap_ufn_setprefix( LDAPObject* self, PyObject* args )
-{
-    char* prefix;
-
-    if (!PyArg_ParseTuple( args, "s", &prefix)) return NULL;
-    if (not_valid(self)) return NULL;
-    ldap_ufn_setprefix( self->ldap, prefix );
-    Py_INCREF( Py_None );
-    return Py_None;
-}
-
-static char doc_ufn[] =
-"ufn_setfilter(filtername) -> None\n"
-"ufn_setprefix(prefix) -> None\n"
-"ufn_search_s(url [,attrsonly=0])\n"
-"ufn_search_st(url [,attrsonly=0 [,timeout=-1]])\n\n"
-"\tSee the LDAP library manual pages for more information on these\n"
-"\t`user-friendly name' functions.";
-
 /* ldap_sort_entries */
 
 /* ldap_url_search */
@@ -1631,9 +1548,6 @@
     {"search",		(PyCFunction)l_ldap_search,		METH_VARARGS,	doc_search},	
     {"search_s",	(PyCFunction)l_ldap_search_st,		METH_VARARGS,	doc_search},	
     {"search_st",	(PyCFunction)l_ldap_search_st,		METH_VARARGS,	doc_search},	
-    {"ufn_search_s",	(PyCFunction)l_ldap_ufn_search_s,	METH_VARARGS,	doc_ufn},
-    {"ufn_setfilter",	(PyCFunction)l_ldap_ufn_setfilter,	METH_VARARGS,	doc_ufn},
-    {"ufn_setprefix",	(PyCFunction)l_ldap_ufn_setprefix,	METH_VARARGS,	doc_ufn},
     {"url_search_s",	(PyCFunction)l_ldap_url_search_st,	METH_VARARGS,	doc_url_search},	
     {"url_search_st",	(PyCFunction)l_ldap_url_search_st,	METH_VARARGS,	doc_url_search},	
     { NULL, NULL }
